<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>679</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>INI editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/img/logo.png</normaloff>:/new/img/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeWidget" name="tree_sect">
          <column>
           <property name="text">
            <string>Section name</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="line_add_new_sect">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>[ ]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_add_new_sect">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>13</height>
             </size>
            </property>
            <property name="text">
             <string>Enter new section</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="but_add_s">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>New</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="but_del_s">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::DotLine</enum>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>10</number>
          </attribute>
          <column>
           <property name="text">
            <string>Param</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="line_add_new_param">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_add_new_param">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>79</width>
              <height>13</height>
             </size>
            </property>
            <property name="text">
             <string>Enter new string</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="but_add">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>New</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="but_del">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionLoad_File"/>
    <addaction name="actionSave_file"/>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_File"/>
   <addaction name="actionSave_file"/>
   <addaction name="actionClear"/>
  </widget>
  <action name="actionSave_file">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/img/save.png</normaloff>:/new/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad_File">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/img/open.png</normaloff>:/new/img/open.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/img/clear.png</normaloff>:/new/img/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
